helm/hpcc/templates/_helpers.tpl

```diff
@@ -453,12 +456,13 @@ The plane will generate a volume if it matches either an includeLabel or an incl
 
 {{/*
 Check that the data plane name is valid, and report an error if not
-Pass in dict with root, planeName
+Pass in dict with root, planeName and optional msg
```

Comment:
minor: msg isn't the best name since it isn't the message, more a categeory/prefix

Reply:
changed to contextPrefix